Corona-Warn-App/src/main/res/layout/tracing_details_item_behavior_row_view.xml

```diff
@@ -40,10 +38,25 @@
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_marginStart="@dimen/spacing_small"
-        android:layout_marginEnd="@dimen/spacing_small"
-        app:layout_constraintEnd_toEndOf="parent"
+        android:layout_marginEnd="@dimen/spacing_tiny"
+        app:layout_constraintEnd_toStartOf="@id/info_icon"
         app:layout_constraintStart_toEndOf="@+id/icon_background"
+        app:layout_goneMarginEnd="@dimen/no_padding"
         app:layout_constraintTop_toTopOf="@+id/icon_background"
         tools:text="@string/risk_details_behavior_body_contact_doctor" />
 
+    <ImageView
+        android:id="@+id/info_icon"
+        android:layout_width="@dimen/spacing_normal"
+        android:layout_height="@dimen/spacing_normal"
+        android:background="?android:attr/selectableItemBackgroundBorderless"
+        android:clickable="true"
+        android:contentDescription="@null"
```

Comment:
You need to add a content description ,because it is a clickable view. See info icon in contact journal items and use text from there

Reply:
updated